queries.php

```diff
@@ -134,7 +134,7 @@
                     </tfoot>
                 </table>
             </div>
-          <button type="button" id="resetButton" hidden="true">Clear Filters</button>
+          <button type="button" id="resetButton" hidden="true" style="color: red; font-weight: bold;">Clear Filters</button>
```

Comment:
Can you move this to `styles/pihole.css` (you can use the `#resetButton` selector)?